src/Uno.UI/UI/Xaml/Media/Animation/Storyboard.cs

```diff
@@ -350,17 +350,9 @@ void ITimelineListener.ChildCompleted(Timeline child)
 			}
 		}
 
-		protected override void Dispose(bool disposing)
```

Comment:
After thinking about it, we probably should be relying on storyboard only to dispose all inner items. Let's keep the `Dispose` methods (but internal) and schedule only once from `Storyboard`.